client/human/HumanClientApp.cpp

```diff
@@ -1212,18 +1212,18 @@ namespace {
     };
 }
 
-void HumanClientApp::ResetToIntro()
-{ ResetOrExitApp(true); }
+void HumanClientApp::ResetToIntro(bool is_error /*= false*/)
```

Comment:
The `/*= false*/` seems misleading since the declaration has no default parameter value.